src/app/about-app/about-app/about-app.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <app-breadcrumb></app-breadcrumb>
+  <header>
+    <p class="font-small font-yellow fw-bold mb-1">{{ "HELPUA" | translate }}</p>
+    <h1 class="header-2 font-darkblue fw-bold">{{ "ABOUT_APP" | translate }}</h1>
+  </header>
+
+  <article>
+    <mat-card class="content-wrapper">
+      <div class="row">
+        <div class="col-sm-12">
+          <p class="title">{{ "ABOUT_APP" | translate }}</p>
+          <p class="text">{{ "ABOUT_APP_DESCRIPTION" | translate }}</p>
+          <p class="title">{{ "PARTICIPANTS" | translate }}</p>
+          <p class="text">
+            {{ "ORDER_BY" | translate }} <br /><br />
+            <a href="https://coi.gov.pl"
+              ><b>{{ "COI" | translate }}</b></a
+            ><br />
+            <a href="https://www.sysopspolska.pl"
+              ><b>{{ "SYSOPS_DEVOPS" | translate }}</b></a
+            ><br />
+            <a href="https://www.snowflake.com"
+              ><b>{{ "SNOWFLAKE" | translate }}</b></a
+            ><br />
+            <a href="https://www.warsawjs.com"
+              ><b>{{ "WARSAW_JS" | translate }}</b></a
+            ><br />
+            <a href="#"
+              ><b>{{ "WEBSCOUTS" | translate }}</b></a
+            ><br />
```

Comment:
It looks like we won't list the orgs so this will need to get updated.

Reply:
z tego co wiem to treść tej zakładki jest cały czas przygotowywana i jak zostanie skonczona to wtedy wpadnie zadanie zeby zaktualizowac